src/io/binary_parser.cpp

```diff
@@ -202,26 +254,26 @@ long long read_edges(string filename, dd::FactorGraph &fg)
         file.read((char *)&position, 8);
         file.read((char *)&padding, 1);
         if (!file.read((char *)&equal_predicate, 8)) break;
-        
+
         variable_id = bswap_64(variable_id);
         factor_id = bswap_64(factor_id);
         position = bswap_64(position);
         ispositive = padding;
         equal_predicate = bswap_64(equal_predicate);
-        
+
         count++;
         // printf("varid=%lli, factorid=%lli, position=%lli, predicate=%lli\n", variable_id, factor_id, position, equal_predicate);
 
         //std::cout << variable_id << std::endl;
 
         // wrong id
-    	if(variable_id >= fg.n_var || variable_id < 0){
-    	  assert(false);
-    	}
+        if(variable_id >= fg.n_var || variable_id < 0){
```

Comment:
It's better not to include these non-code changes (space changes, empty lines, etc.). If we would like to later backtrack the history of the code, it might cause some confusion (track the most recent changes, etc.).